### PR TITLE
Add the ability for some enums to still deserialize properly

### PR DESCRIPTION
**Pull request by brentryan**
See https://github.com/dropwizard/dropwizard/pull/1220 for more details.

### CODE REVIEW

File: dropwizard-jackson/src/main/java/io/dropwizard/jackson/FuzzyEnumModule.java [hunk 1]
```diff
@@ -48,22 +50,33 @@ protected PermissiveEnumDeserializer(Class<Enum<?>> clazz) {
...
             throw ctxt.mappingException(text + " was not one of " + acceptedValues);
         }
     }
 
     private static class PermissiveEnumDeserializers extends Deserializers.Base {
+
         @Override
         @SuppressWarnings("unchecked")
         public JsonDeserializer<?> findEnumDeserializer(Class<?> type,
-                                                        DeserializationConfig config,
```
Comment (joschi) [inline dropwizard-jackson/src/main/java/io/dropwizard/jackson/FuzzyEnumModule.java]: Please don't add unnecessary whitespace changes.

**Comment by brentryan** [comment]
Follow changes here https://github.com/dropwizard/dropwizard/pull/1340
